Makefile

```diff
@@ -1,4 +1,4 @@
-export DOCKER_IMAGE ?= cloudposse/bastion
+export DOCKER_IMAGE ?= cloudposse/$(APP)
```

Comment:
It can be explicitly defined for anyone who wants to either in the makefile or on the command line (e.g. `make docker:build APP=foobar`). What I don't like is that there've been multiple copy & paste errors like this.